components/user_settings/advanced/user_settings_advanced.jsx

```diff
@@ -332,8 +343,8 @@ export default class AdvancedSettingsDisplay extends React.PureComponent {
                 <fieldset key='ctrlSendSetting'>
                     <legend className='form-legend hidden-label'>
                         <FormattedMessage
-                            id='user.settings.advance.sendTitle'
-                            defaultMessage='Send messages on CTRL+ENTER'
+                            id={fmIdForUserSettingsAdvSendTitle}
+                            defaultMessage={defaultMessageSendTitle}
```

Comment:
Then this can be ``<FormattedMessage {...ctrlSendTitle}/>`` for title and ``<FormattedMessage {...ctrlSendDesc}/>`` for description.